src/main/webapp/history/history.js

```diff
@@ -19,51 +18,337 @@
                 tooltip.enable();
             });
         });
+		
+		function renderTrendChart(chartDivId, model, settingsDialogId, chartClickedEventHandler) {
+			const chartPlaceHolder = document.getElementById(chartDivId);
+			const chart = echarts.init(chartPlaceHolder);
+			chartPlaceHolder.echart = chart;
+			
+			const textColor = getComputedStyle(document.body).getPropertyValue('--darkreader-text--text-color') || getComputedStyle(document.body).getPropertyValue('--text-color') || '#222';
+			const showSettings = document.getElementById(settingsDialogId);
+			let darkMode = getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background')
+			darkMode = darkMode !== undefined && darkMode !== null && darkMode !== ''
+			console.log('darkMode: ' + darkMode)
+			let series = model.duration.series.concat(model.result.series)
+			series.forEach(s => s.emphasis = {
+				disabled: true
+			});
+			const options = {
+				animation: false,
+				darkMode: darkMode,
+				//backgroundColor: getComputedStyle(document.body).getPropertyValue('--darkreader-bg--background') || getComputedStyle(document.body).getPropertyValue('--bs-body-bg') || '#fff',
+				toolbox: {
+					feature: {
+					  dataZoom: {
+						yAxisIndex: 'none'
+					  },
+					  restore: {},
+					  saveAsImage: {}
+					}
+				},
+				tooltip: {
+					trigger: 'axis',
+					animation: false,
+					axisPointer: {
+						type: 'cross',
+						label: {
+							backgroundColor: '#6a7985'
+						},
+						animation: false
+					},
+					transitionDuration: 0,
+					textStyle: {
+						fontSize: 12,
+					},
+					padding: 5,
+					order: 'seriesAsc',
+					position: [-400, '7%'],
+				},
+				axisPointer: {
+					snap: true,
+					link: [
+					  {
+						xAxisIndex: 'all'
+					  }
+					]
+				  },
+				dataZoom: [
```

Comment:
the zoom and zoom reset buttons don't seem to do anything for me, only the reset and download as image do things 

<img width="78" alt="image" src="https://github.com/jenkinsci/junit-plugin/assets/21194782/89d9ae7d-1ead-407b-9685-bc0dc4941ad6">